app/renderer/js/utils/domain-util.js

```diff
@@ -20,7 +20,7 @@ const logger = new Logger({
 
 let instance = null;
 
-const defaultIconUrl = '../renderer/img/icon.png';
+const defaultIconUrl = __dirname.slice(0, -9) + '/img/icon.png';
```

Comment:
@flowkraD can you document why you are using the `(0, -9)` range?

Reply:
Can I add a comment above it?

Reply:
I added a comment for now. Any changes or addition required can be mentioned and will be implemented at my earliest convenience.
Also i squashed the two commits - the changes to the code and the one with comments - into one.
Being new to Open-Source, I will highly value any advice you give me. Thank you (^_^)

Reply:
It does feel like a unthoughtful hack but using `__dirname` and slicing it, we get the whole path and one that is Independent of where you are accessing it from. You can refer to the `defaultIconUrl` from anywhere in the code base and it points to the same image. I thought this way would make it bullet proof for future in case this path is referred at any place that isn't coved by declaring it as just `../../img/icon.png`
I you know of any better way, I would love to know and make changes accordingly (^_^)